### ISSUE TITLE
Basic search

### ISSUE THREAD

**Issue by istride** [unicef/iogt#57]
Users should be able to search for content.

Users can enter search terms in input box in the site header.
Users are shown a page of results.
Search index should use Elasticsearch as a backend.

**Comment by ehmadzubair** [comment]
@istride other than the search functionality of the ticket, I'm planning to add a `docker-compose.yml` file that spins up the project and an Elasticsearch container. This will help us keep the initial setup simple. Thoughts?

**Comment by isedwards** [comment]
@ehmadzubair can we keep Elasticsearch as an optional capability so that it is easy to deploy to dev/test without it when required?

**Comment by istride** [comment]
@ehmadzubair yes, I think docker compose would be useful for replicating a production-like configuration in development.
